google/datalab/kernel/__init__.py

```diff
@@ -30,6 +30,7 @@
 # Import the modules that do cell magics.
 import google.datalab.bigquery.commands
 import google.datalab.commands
+import google.datalab.contrib.pipeline.commands
```

Comment:
This line needs to be removed so %%pipeline is not automatically loaded, just as %%ml.